Components/Hlms/Pbs/src/Cubemaps/OgreCubemapProbe.cpp

```diff
@@ -380,6 +388,23 @@ namespace Ogre
         if( workspaceDefOverride != IdString() )
             workspaceDef = compositorManager->getWorkspaceDefinition( workspaceDefOverride );
 
+        const CompositorWorkspaceDef::NodeAliasMap& workspaceNodes = workspaceDef->getNodeAliasMap();
+        assert( workspaceNodes.size() == 1 );
+
+        CompositorNodeDef* probeNode = compositorManager->getNodeDefinitionNonConst( workspaceNodes.begin()->first );
+        assert( probeNode->getNumTargetPasses() % 6 == 0 );
+        for( size_t i=0; i<probeNode->getNumTargetPasses(); ++i )
+        {
+            CompositorTargetDef* probeTarget = probeNode->getTargetPass(i);
+
+            const CompositorPassDefVec& probePasses = probeTarget->getCompositorPasses();
+            for( size_t j=0; j<probePasses.size(); ++j )
+            {
+                probePasses[j]->mExecutionMask = ( ExecutionFlags::FIRST_SLICE_EXECUTION_FLAG << probeTarget->getRtIndex() ) |
+                                                 probePasses[j]->mExecutionMask & ~ExecutionFlags::NO_SLICE_EXECUTION_FLAG;
```

Comment:
Turn on the `ExecutionFlags::FIRST_SLICE_EXECUTION_FLAG` bit (shifted left by _n_ positions where _n_ is the cubemap face index) and turn off the `ExecutionFlags::NO_SLICE_EXECUTION_FLAG` bit

Reply:
Code has slightly changed:
```
probePasses[j]->mExecutionMask &= ExecutionFlags::RESERVED_EXECUTION_FLAGS;
if( i == probeNode->getNumTargetPasses() - 1 )
	probePasses[j]->mExecutionMask |= ExecutionFlags::TS_IBL_EXECUTION_FLAG;
else
	probePasses[j]->mExecutionMask |= ExecutionFlags::TS_FIRST_FACE_EXECUTION_FLAG << probeTarget->getRtIndex();
```
Turn on the `ExecutionFlags::TS_FIRST_FACE_EXECUTION_FLAG` bit (shifted left by _n_ positions where _n_ is the cubemap face index) or the `ExecutionFlags::TS_IBL_EXECUTION_FLAG` bit and  combine it with `mExecutionMask `.